modules/flight_interface/flight_interface_worker.py

```diff
@@ -45,4 +51,11 @@ def flight_interface_worker(
         if not result:
             continue
 
+        # Replace any existing odometry data with the latest odometry data
+        try:
+            odometry_queue.queue.get_nowait()
+        except queue_proxy_wrapper.queue.Empty:
```

Comment:
Import `queue` standard library and use that instead of using a transitive dependency.